iquip/apps/builder.py

```diff
@@ -404,6 +478,33 @@ def initSchedOptsEntry(self):
             self.builderFrame.schedOptsListWidget.addItem(item)
             self.builderFrame.schedOptsListWidget.setItemWidget(item, widget)
 
+    @pyqtSlot()
+    def reloadArgs(self):
+        """Reloads the build arguments.
+        
+        Once the reloadArgsButton is clicked, this is called.
+        """
+        self.thread = ExperimentInfoThread(self.experimentPath, self.onReloaded, self)
```

Comment:
All the public attributes should be defined in `__init__()` and documented.

Reply:
Oh, we didn't document `self.thread` in `explorer.py`, too.

Can I do this here, or is it better to make it as another issue?

Reply:
I think you can do it here!